twill-core/src/main/java/org/apache/twill/internal/Configs.java

```diff
@@ -31,6 +31,11 @@
      */
     public static final String JAVA_RESERVED_MEMORY_MB = "twill.java.reserved.memory.mb";
 
+    /**
+     * Set this to false to disable the secure store updates done by default.
+     */
+    public static final String ENABLE_DEFAULT_SECURE_STORE_UPDATE = "twill.secure.store.update.location.enabled";
```

Comment:
Updated and squashed.
This was the only diff:
![image](https://cloud.githubusercontent.com/assets/2440977/18075464/eab7cfa4-6e29-11e6-8373-a70e47fcb620.png)